src/helper_search.py

```diff
@@ -1,20 +1,36 @@
 #!/usr/bin/python2.7
+"""
+src/helper_search.py
+====================
 
-from helper_sql import *
+"""
+
+from helper_sql import sqlQuery
 
 try:
     from PyQt4 import QtGui
     haveQt = True
 except ImportError:
     haveQt = False
 
-def search_translate (context, text):
+
+def search_translate(context, text):
```

Comment:
Did you try `from tr import _translate`?

Reply:
This was autopep8 correctly removing the space before the parentheses. Unless you meant to specify a different line?